packages/outputarea/src/widget.ts

```diff
@@ -1045,7 +1045,7 @@ namespace Private {
     renderModel(model: IRenderMime.IMimeModel): Promise<void> {
       return this._wrapped.renderModel(model).then(() => {
         const win = (this.node as HTMLIFrameElement).contentWindow;
-        if (win) {
+        if (win && win.location.href != window.location.href) {
           win.location.reload();
         }
```

Comment:
@blink1073 what was the reason for this reload in the first place? It seems that my condition disables it completely but the flag example still works. Could we remove it altogether, or is it needed for some other use case?

Reply:
I can't remember that far back ;).  I'd say as long as this works in the three supported browsers we are fine.

Reply:
I tested the binder link.  Looks good to me on recent version of all three browsers, thanks!